### PR TITLE
replace akka based names in logging code

### PR DESCRIPTION
**Pull request by pjfanning**
splits out some of the changes in #184 so that can focus more on the Pekko/Akka URL protocol

### CODE REVIEW

File: actor-tests/src/test/scala/org/apache/pekko/serialization/SerializeSpec.scala [hunk 1]
```diff
@@ -276,7 +276,7 @@ class SerializeSpec extends PekkoSpec(SerializationTests.serializeConf) {
     }
 
     "log warning if non-Pekko serializer is configured for Pekko message" in {
-      EventFilter.warning(pattern = ".*not implemented by Apache Pekko.*", occurrences = 1).intercept {
+      EventFilter.warning(pattern = ".*not implemented by Pekko.*", occurrences = 1).intercept {
```
Comment (jrudolph) [inline actor-tests/src/test/scala/org/apache/pekko/serialization/SerializeSpec.scala:279]: Weird, a reverse change like this is also proposed as part of #196?

**Review by jrudolph — APPROVED** [review]
I don't mind the change but it looks as if it would conflict with #196?

**Comment by pjfanning** [comment]
> 196

I was trying to fix an issue in #196 and committed the change to this PR by mistake. I committed a revert of that commit to this PR after committing the fix to #196. I am getting no conflicts showing for this PR now that #196 is merged.

**Comment by pjfanning** [comment]
The squash merge led to this commit: https://github.com/apache/incubator-pekko/commit/2824a44abeedaefffc68f4930d5f2435690d1eb2 

There appears to be no change to SerializeSpec in that commit.

**Pull request merged**